ext_localconf.php

```diff
@@ -1,8 +1,6 @@
 <?php
 defined('TYPO3_MODE') or die();
 
-$GLOBALS['TYPO3_CONF_VARS']['SC_OPTIONS']['GLOBAL']['softRefParser']['rtehtmlarea_images'] = \Netresearch\RteCKEditorImage\Database\RteImagesSoftReferenceIndex::class;
```

Comment:
the rst file states that having both the global as well as the services.yaml reference should work - however, I get an error about circular references then.